RELEASE_NOTES.md

```diff
@@ -1,5 +1,11 @@
 # Moirai Release Notes
 
+## Moirai 2.0.0
+
+* When using a `ProportionOfUsersConfigDecider`, the hash will be based on the feature identifier and userId instead of just the userId.
+  * This makes sure that different features don't allows target the same set of users.
```

Comment:
This sentence is off.

Reply:
Whoops, somehow said "allows" instead of "always". Fixed.